toolchain/check/testdata/interop/cpp/macros.carbon

```diff
@@ -254,28 +254,101 @@ fn F() {
 }
 
 // --- string_literal_object_like_macro.h
-#define CONFIG_VALUE "abc"
 
-// --- fail_todo_import_string_literal_object_like_macro.carbon
+#define SimpleString "abc"
+#define EmptyString ""
+#define EscapeCharacter " \t "
+#define Concatenated "a" "b"
```

Comment:
Maybe also test indirecting through another macro? And specifically some fancier thing like:

```cpp
#define CONCAT_STR(A, B) #A #B
#define Indirect CONCAT(x, y)
```

And imports `Indirect` successfully?